api/v1/views/states.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+""" Handle RESTful API actions for State objects """
+from api.v1.views import app_views
+from flask import abort, request, jsonify, make_response
+from models import storage
+from models.state import State
+
+
+@app_views.route('/states', methods=['GET'], strict_slashes=False)
+def states_objects():
+    """ Retrieves the list of all State objects """
+    state_objs = storage.all(State)
+    state_json = []
+    for obj in state_objs:
```

Comment:
I think this line can be `for obj in state_objs.values()` because we want to use the value not the key

Reply:
let me correct this.